drivers/storage/portworx/portworx.go

```diff
@@ -203,6 +203,13 @@ func (p *portworx) SetDefaultsOnStorageCluster(toUpdate *corev1.StorageCluster)
 			toUpdate.Status.DesiredImages.UserInterface = release.Components.Lighthouse
 		}
 
+		if autoUpdateTelemetry(toUpdate) &&
```

Comment:
Can we add UTs for this part similar to what we have for other components? You can use this for reference - https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/portworx_test.go#L507?

Reply:
added